syntax = "proto3";

package CSR;

message CSTBS{
	int64 deviceID = 1;
	int64 orgID = 2;
	string curveID = 3;
    string hashID = 4;
    int64 pubKLen = 5;
}	